sources/packages/web/src/components/common/IdleTimeChecker.vue

```diff
@@ -32,22 +46,35 @@ export default {
     },
   },
   setup(props: any) {
-    const lastActivityLogin = ref(new Date());
     const interval = ref();
     const extendTimeModal = ref({} as ModalDialog<boolean>);
     const { isAuthenticated } = useInstitutionAuth();
     const { getDatesDiff } = useFormatters();
+    const { executeLogout } = useAuth();
+    const countdown = ref(COUNT_DOWN_TIMER_FOR_LOGOUT);
+    const countdownInterval = ref(0);
```

Comment:
I do not see a reason for the `countdownInterval` be declared with `red`. Am I missing something?